pkg/action/lazyclient.go

```diff
@@ -116,6 +117,13 @@ func (s *secretClient) Patch(ctx context.Context, name string, pt types.PatchTyp
 	return s.client.CoreV1().Secrets(s.namespace).Patch(ctx, name, pt, data, opts, subresources...)
 }
 
+func (s *secretClient) Apply(ctx context.Context, secretConfiguration *applycorev1.SecretApplyConfiguration, opts metav1.ApplyOptions) (*v1.Secret, error) {
```

Comment:
I'm assuming these are new functions introduced to an interface somewhere. Do you have a link handy to the changes so we can investigate further?

Reply:
https://github.com/helm/helm/blob/257a427866883a9fb745be40800ffe8e9ceda82b/pkg/action/lazyclient.go#L58

This line caught the changes :)